charts/thoras/values.yaml

```diff
@@ -71,20 +71,10 @@ metricsCollector:
     imageTag: "2.17.2-pg16"
     name: timescale
     containerPort: 5432
+  blobService:
+    containerPort: 80
```

Comment:
[somewhere-between-pebble-and-boulder] See [this comment](https://github.com/thoras-ai/helm-charts/pull/208/files#r2008016296)

It's great if the service is 80 but we should have have containers themselves listen on unprivileged ports